Telegram/README.md

```diff
@@ -0,0 +1,46 @@
+# Telegram Integration Prototype
```

Comment:
The current folder structure/naming might cause confusions with other ES integrations. I guess we could separate them. It might be better if there are `es` and `cx` folders at top level, and then the integration sub-folders in them (e.g.,  cx/telegram).

nit: probably also change the folder naming style to be consistent with the current convention (e.g., `dialogflow-api-quick-start`).